weed/shell/command_volume_move.go

```diff
@@ -55,6 +55,7 @@ func (c *commandVolumeMove) Do(args []string, commandEnv *CommandEnv, writer io.
 	sourceNodeStr := volMoveCommand.String("source", "", "the source volume server <host>:<port>")
 	targetNodeStr := volMoveCommand.String("target", "", "the target volume server <host>:<port>")
 	diskTypeStr := volMoveCommand.String("disk", "", "[hdd|ssd|<tag>] hard drive or solid state drive or any tag")
+	compactionBytePerSecond := volMoveCommand.Int64("compactionBytePerSecond", 0, "limit the speed of copy")
```

Comment:
this is not compaction. how about `ioBytesPerSecond`?

Reply:
I agree with you.